README.md

```diff
@@ -521,19 +521,31 @@ $pinny.pinny('close');
 
 Currently, form inputs and selects inside of Pinny have issues on iOS7 and under. This is due to not being able to lock scrolling without causing rendering issues as well as iOS attempting to scroll the page when the keyboard opens. Forms work but will cause some visual jumping.
 
-## Building a distribution
+## Working with Pinny locally
 
 ### Requirements
-* [node.js 0.10.x/npm](http://nodejs.org/download/)
+
+* [Node.js 0.10.x/npm](http://nodejs.org/download/)
 * [Grunt](http://gruntjs.com/)
-    * Install with `npm install -g grunt-cli`
+  * Install with `npm install -g grunt-cli`
 * [Bower](http://bower.io/)
-    * Install with `npm install -g bower`
+  * Install with `npm install -g bower`
+
 
 ### Steps
 1. `npm install`
 1. `bower install`
-1. `grunt build`
+1. `grunt serve`
+
+
+##### Grunt Tasks:
```

Comment:
This can probably be just a level 3 heading, since I don't think it makes sense to be a sub-topic of "Steps"